### PR TITLE
tlon.io: new styling, blog, etc.

### PR DESCRIPTION
**Pull request by urcades**
Part 2 of 2 of an update to tlon.io, using a new styling approach, and generally restructuring the repo.

Good morning @matildepark — I'm running into a really weird issue that I suspect is derived from zola's templating scheme where my css is not being reflected on galen's post located in News. The css appears to be linked correctly otherwise on other pages, but for some reason is being fetched as such, as a path based off the blog itself.

Screen Shot 2021-01-27 at 11 14 57 AM

No idea why this is happening, any insight would be welcome! 

Also, the usual span of checkups/sanity checks are needed.

Todos:

- [x] Figure out how to surface the post's unique metadata (author, patP, etc.)
- [x] Finish up the assets in galen's blog post
- [x] Update the hero asset depicting mobile + desktop Landscape
- [ ] Correctly set up SEO-related elements to the site

### CODE REVIEW

**Comment by urcades** [comment]
The Big Note to make with this refresh, if it isn't immediately clear to the person delving into the markdown, is that it is now using Tailwind CSS, which is more or less executed exactly the same manner we used Tachyons, but seemingly much more robust, and still being currently developed (Tachyons has halted its development for nearly a year).

The issues I outlined running into above seem more to be Zola-based templating issues, but in the off chance you need to muck around with styling for any reason, Tailwind's docs are wonderful:

https://tailwindcss.com/docs/container

**Comment by matildepark** [comment]
> Good morning @matildepark — I'm running into a really weird issue that I suspect is derived from zola's templating scheme where my css is not being reflected on galen's post located in News. The css appears to be linked correctly otherwise on other pages, but for some reason is being fetched as such, as a path based off the blog itself.

base.html had it imported as `style.css`, which is relative to current directory. `/style.css` is relative to site root. Fixed.

>  Figure out how to surface the post's unique metadata (author, patP, etc.)

Surface the metadata in the markdown under `[extra]` like we do on urbit.org. Then on the posts template you can do

```
{ if page.extra.author }
<span> {{page.extra.author}} </span>
{ endif }
```

Pseudocode but you get it. Not that bad.

**Comment by urcades** [comment]
This appears to be complete and ready to merge.

I think this may need an SEO pass though

**Comment by matildepark** [comment]
Maybe a minified tailwind might be preferable.

**Comment by urcades** [comment]
I can hack that once I fold in some updates GWP suggested!

—
~fabled-faster
https://urbit.org/

**Comment by urcades** [comment]
@matildepark reading through tailwind's docs, which seem to lean quite heavily on using a thing called PostCSS to handle CSS minification: https://tailwindcss.com/docs/using-with-preprocessors

I'm finding the config a little hard to grok as instantly as I'd like it to, and I'd like to move on to some other works that need tending to.

Unless this is something you think absolutely needs to be done before this is merged, this is effectively ready to go at this point

**Comment by matildepark** [comment]
I'd [use the CDN or host its minified CSS on the repo here](https://tailwindcss.com/docs/installation#using-tailwind-via-cdn). It's just checking in like 140,000 lines...

**Comment by urcades** [comment]
lmao, okay — great call, will edit the line , and merge accordingly.

very exciting times ...

—
~fabled-faster
https://urbit.org/

File: content/news/20210123-landscape-now.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
++++
+title = 'Landscape, Now'
+description = 'Landscape does exactly what it appears to do and nothing else. There are no ads or tracking, there are no gimmicks to try to keep you engaged or addicted, and there’s no quiet uncertainty that Landscape might disappear or change drastically.'
+date = 2019-11-27
```
Comment (matildepark) [inline content/news/20210123-landscape-now.md]: This date appears to be Ancient? Is this that old?

**Comment by matildepark** [comment]
Finally you still have to a actually remove the tailwind.scss file.

File: content/news/20210123-landscape-now.md (hunk 1 above)
Comment (urcades) [reply content/news/20210123-landscape-now.md]: Oh yeah, jfc

**Review by matildepark — APPROVED** [review]

**Pull request merged**